### PR TITLE
Use https for all links

### PR DESCRIPTION
**Pull request by dns2utf8**
Hi all

This is a followup to https://github.com/gtk-rs/gtk/pull/1054

Cheers,
Stefan

### CODE REVIEW

File: docs-src/tutorial/cross.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The mingw packages are in the AUR, you can either install manually or use a help
 
 If you can't find gtk precompiled dll, you can follow these steps:
 
-1. Download the 64 bit version of the latest precompiled gtk libraries from [here](http://win32builder.gnome.org/).
+1. Download the 64 bit version of the latest precompiled gtk libraries from [here](https://win32builder.gnome.org/).
```
Comment (GuillaumeGomez) [inline docs-src/tutorial/cross.md:33]: Hum, this link is dead. ping @sdroege

**Comment by GuillaumeGomez** [comment]
Thanks!

File: docs-src/tutorial/cross.md (hunk 1 above)
Comment (dns2utf8) [reply docs-src/tutorial/cross.md:33]: There are some old packages in the WebArchive:
https://web.archive.org/web/20180719181100/http://win32builder.gnome.org/
I assume there is a newer way for windows users?

Comment (sdroege) [reply docs-src/tutorial/cross.md:33]: > Hum, this link is dead. ping @sdroege

I don't know, what's that? For cross-compiling, Arch has [this](https://aur.archlinux.org/packages/mingw-w64-gtk3/) and Fedora also has packages. Ubuntu/Debian don't IIRC.

Comment (dns2utf8) [reply docs-src/tutorial/cross.md:33]: I assume it is for corss-building windows applications

**Pull request merged**